acquisitions/src/main/resources/thunderjet/cross-modules/features/check-encumbrances-after-order-line-exchange-rate-update.feature

```diff
@@ -0,0 +1,237 @@
+# For MODORDERS-1185
+Feature: Check encumbrances after order line exchange rate update
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+    * call loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * call loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def createInvoice = read('classpath:thunderjet/mod-invoice/reusable/create-invoice.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
+
+  @Positive
+  Scenario: Check encumbrances after order line exchange rate update to manual rate 2 times
+    * def fundId = call uuid
+    * def budgetId = call uuid
+    * def orderId = call uuid
+    * def poLineId = call uuid
+    * def invoiceId = call uuid
+    * def invoiceLineId = call uuid
+
+    ### 1. Create finances
+    * configure headers = headersAdmin
+    * def v = call createFund { id: "#(fundId)" }
+    * def v = call createBudget { id: "#(budgetId)", allocated: 10000, fundId: "#(fundId)", status: "Active" }
+    * configure headers = headersUser
```

Comment:
as a option: we can extract these duplicate prep fund and budget to one scenario by naming create fund and budget since both are the same. https://github.com/folio-org/folio-integration-tests/pull/1534/files#diff-c7ab6f3c8f5642fa693b93d6478fbefafb677baefefdf69ac5112f423c8c173bR141-R146

Reply:
> as a option: we can extract these duplicate prep fund and budget to one scenario by naming create fund and budget since both are the same. https://github.com/folio-org/folio-integration-tests/pull/1534/files#diff-c7ab6f3c8f5642fa693b93d6478fbefafb677baefefdf69ac5112f423c8c173bR141-R146

I think this bit is already well optimized, both call reusable functions

Reply:
if you consider to use this approach, you should move budget, fundId uuid to top and change to 
callonce uuid1 and callonce uuid2

Reply:
> if you consider to use this approach, you should move budget, fundId uuid to top and change to callonce uuid1 and callonce uuid2

These tests require new budgets as we are asserting encumbrance transactions tied to the fund & budget, if we move them up then the calculation has to change

Reply:
Make sense, I wrote second comment with same time with your first comment, so you can ignore, it is option, small suggestion style comment